src/frontend/js/delivery.js

```diff
@@ -7,4 +7,12 @@ $(function() {
         window.location.replace($url+value);
     });
 
+    $('.button.orders').click(function(){
+        $('.button.orders i').addClass('loading');
+        var url = $(this).attr('data-url');
+        $.get(url, function( data ) {
+            window.location.reload();
```

Comment:
@manumilou, I don't get this.. we make an ajax call to generate the orders to then reload the page? Why not send the orders created in the AJAX response and update the DOM instead recharging the page and make another request to the server?